app/src/main/java/fr/acinq/phoenix/utils/Wallet.kt

```diff
@@ -133,17 +134,24 @@ object Wallet {
     val conf = HashMap<String, Any>()
 
     // electrum config
-    val electrumServer = Prefs.getElectrumServer(context)
-    if (!Strings.isNullOrEmpty(electrumServer)) {
+    val electrumServerAndProtocol = Prefs.getElectrumServer(context)
+    if (!Strings.isNullOrEmpty(electrumServerAndProtocol)) {
       try {
+        val ssl = when {
+          electrumServerAndProtocol.startsWith("ssl://") -> "strict" // custom server certificate must be valid
+          electrumServerAndProtocol.startsWith("tcp://") -> "off"
+          else -> throw IllegalArgumentException("invalid electrum uri")
```

Comment:
It seems a bit harsh to throw an exception here if the uri does not have a protocol. I'm pretty sure that most users now using this feature have not entered a protocol, so this change would break their app (with a startup failure).

Could we fallback to ssl? It would be the same behaviour as what we have now.

Also this means that the app would support connecting to servers with self signed certificates, right?